### ISSUE TITLE
Hippocampal microinfarct

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7046]
**Preferred term label:**

Hippocampal microinfarct

**Synonyms**

**Definition (free text, please give PubMed ID)**

An infarct located in the structure of the cerebrum involved with memory storage and spatial navigation that can only be observed microscopically.
PMID:10867785

**Parent term (use hpo.jax.org/app)**

abnormal hippocampus morphology 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Alzheimer's Disease (OMIM: 606889)

**Your nano-attribution (ORCID)**

https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
thanks @sarahstofel added this term. The PMID does not specifically describe hippocampal microinfarcts in Alzheimer -- this is what we like to have to add disease annotations. If you have a PMID that documents this please send it!

**Comment by pnrobinson** [comment]
We will leave the question of annotations open. The term is valid and was added.
